ash/src/extensions/khr/maintenance4.rs

```diff
@@ -0,0 +1,88 @@
+use crate::vk;
+use crate::{Device, Instance};
+use std::ffi::CStr;
+use std::mem;
+
+#[derive(Clone)]
+pub struct Maintenance4 {
+    handle: vk::Device,
+    fns: vk::KhrMaintenance4Fn,
+}
+
+impl Maintenance4 {
+    pub fn new(instance: &Instance, device: &Device) -> Self {
+        let fns = vk::KhrMaintenance4Fn::load(|name| unsafe {
```

Comment:
please factorize  `device.handle()` like here : https://github.com/MaikKlein/ash/pull/494/files#diff-6eb43a992bec48c8676fdb7916ad74eb423692b90b465dfa8f44717e081d47d8

Reply:
This has been merged long before your PR - in fact you even factorized this out in https://github.com/MaikKlein/ash/pull/494/files#diff-42685c200f5a13446e740816683a82dc8df0dcad950ed9a1b859295870f1b2c1R14-R18 already :)

Reply:
Oops. Sorry for the noise.